includes/class-webmention-receiver.php

```diff
@@ -711,24 +715,66 @@ public static function publish_post_hook( $post_id ) {
 	 * @param WP_Error $error
 	 */
 	public static function delete( $error ) {
-		$error_codes = apply_filters( 'webmention_supported_delete_codes', array(
-			'resource_not_found',
-			'resource_deleted',
-			'resource_removed',
-		) );
+		$error_codes = apply_filters(
+			'webmention_supported_delete_codes', array(
+				'resource_not_found',
+				'resource_deleted',
+				'resource_removed',
+			)
+		);
 
 		if ( ! in_array( $error->get_error_code(), $error_codes ) ) {
 			return;
 		}
 
 		$commentdata = $error->get_error_data();
-		$commentdata = self::check_dupes($commentdata);
+		$commentdata = self::check_dupes( $commentdata );
 
 		if ( isset( $commentdata['comment_ID'] ) ) {
 			wp_delete_comment( $commentdata['comment_ID'] );
 		}
 	}
 
+
+	public static function whitelist_approved( $commentdata ) {
```

Comment:
we discussed a lot of whitelists/blacklists lately (for example a blacklist, to disable sending webmentions to a specific domain https://github.com/pfefferle/wordpress-webmention/issues/166), so we should make the functions more descriptive. At least we should add something like `receiver` to the name.